src/app/components/QuestionAnswers.js

```diff
@@ -0,0 +1,36 @@
+class QuestionAnswers {
+  constructor(id, answers, correctAnswer) {
+    this.render(id, answers);
+    this.correctAnswer = correctAnswer;
```

Comment:
Zamieńmy to kolejnością, funkcja renderContent używa this.correctAnswer, to może powodować problemy, generalnie zawsze staramy się dać render jak najpóźniej